eng/Versions.props

```diff
@@ -29,7 +29,14 @@
     <!-- In order tests against the same version of NuGet as the SDK. We have to set this to match. -->
     <NuGetVersion>6.6.0-preview.3.61</NuGetVersion>
     <!-- roslyn -->
-    <MicrosoftNetCompilersToolsetPackageVersion>4.7.0-1.23226.15</MicrosoftNetCompilersToolsetPackageVersion>
+    <MicrosoftCodeAnalysisCSharpFeaturesVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisCSharpFeaturesVersion>
+    <MicrosoftCodeAnalysisCSharpVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisCSharpVersion>
+    <MicrosoftCodeAnalysisFeaturesVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisFeaturesVersion>
+    <MicrosoftCodeAnalysisVisualBasicFeaturesVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisVisualBasicFeaturesVersion>
+    <MicrosoftCodeAnalysisVisualBasicWorkspacesVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisVisualBasicWorkspacesVersion>
+    <MicrosoftCodeAnalysisVisualBasicVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisVisualBasicVersion>
+    <MicrosoftCodeAnalysisWorkspacesMSBuildVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisWorkspacesMSBuildVersion>
+    <MicrosoftCodeAnalysisVersion>4.7.0-1.23226.15</MicrosoftCodeAnalysisVersion>
```

Comment:
As these are all new versions which presumably aren't used in a non source build build, how do we communicate their intent? Should we add a comment? Will they show up as prebuilt failures if being removed?

Reply:
These are not source-build specific.  Previously the repo was using a single dependency on Microsoft.Net.Compilers.Toolset to represent all of the dependencies from roslyn.  I broken them apart to reference the individual packages.  This felt like the right thing to do based on the questions asked previously about what the Microsoft.Net.Compilers.Toolset dependency was used for.  I don't see a comment as necessary as it is the standard arcade dependency pattern.

Reply:
For clarity, these are all being referenced in https://github.com/dotnet/format/pull/1821/files#diff-5baf5f9e448ad54ab25a091adee0da05d4d228481c9200518fcb1b53a65d4156

Reply:
Ah cool, I missed that. Looks good then.